### ISSUE TITLE
Assert fails when cross compiling

### ISSUE THREAD

**Issue by ChrisGreenaway** [MichaelMcDonnell/test_bin#3]
I received this error when cross compiling to musl:

```
thread 'testing' panicked at 'assertion failed: current_dir.ends_with(\"target/debug/deps\") ||\n    current_dir.ends_with(\"target/release/deps\")', /cargo/registry/src/github.com-1ecc6299db9ec823/test_bin-0.3.0/src/lib.rs:59:5
```

I ran `cross test --release --target x86_64-unknown-linux-musl`.

The binary was in `target/x86_64-unknown-linux-musl/release`.

**Comment by MichaelMcDonnell** [comment]
Thanks for reporting the issue! I can reproduced on my machine.

I'll take a look at supporting cross compiling. Patches are welcome :-)

**Comment by ChrisGreenaway** [comment]
One solution might be to remove the assert completely. Alternatively, changing the assert to just check that the final part of the path is `deps`:
```
assert!(current_dir.ends_with("deps"))
```

I could raise PRs for either of those, if you like.

**Comment by MichaelMcDonnell** [comment]
Let's remove the assert. We can always add back a better assert later if needed. I would appreciate a PR for that.

**Comment by MichaelMcDonnell** [comment]
Thanks again! Your fix was merged in #4. I'll try to get a new release out today.

**Comment by MichaelMcDonnell** [comment]
I've published version 0.4.0.

**Comment by ChrisGreenaway** [comment]
Works great! Thanks for turning this round so quickly.
